particle/converters/pythia.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018-2019, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+
+from __future__ import absolute_import
+
+import csv
+
+from .. import data
+from ..exceptions import MatchingIDNotFound
+from ..pdgid import PDGID
+from ..pythia import PythiaID
+
+
+class __Pythia2PDGIDBiMap(object):
+    """
+    Bi-bidirectional map between PDG and Pythia IDs.
+
+    Examples
+    --------
+    >>> pyid = Pythia2PDGIDBiMap[PDGID(9010221)]
+    >>> pyid
+    <PythiaID: 10221>
+
+    >>> pdgid = Pythia2PDGIDBiMap[PythiaID(10221)]
+    >>> pdgid
+    <PDGID: 9010221>
+    """
+    def __init__(self):
```

Comment:
I would generalize this. Allow a user to pick two classes; they become A and B, and then they are selected in the `__getitem__`.

Reply:
PS: I can make this change if you want me to in 1-2 days.

Reply:
Good idea. I will try and get to this this week, then, no worries ...

Reply:
OK, @henryiii, I've just committed a generic bi-map class. Have a look. I can then trivially define `Pythia2PDGIDBiMap` as function of it.